packages/util/tests/dummy/app/index.html

```diff
@@ -17,7 +17,7 @@
     {{content-for "body"}}
 
     <script src="/@embroider/core/vendor.js"></script>
-    <script src="/@embroider/core/entrypoint" type="module"></script>
+    <script src="/@embroider/virtual/compat-modules" type="module"></script>
```

Comment:
this will literally do nothing 🤔 are these tests even run?

Reply:
Better to be updated than to lie?

🤷🙈🙃

Reply:
I mean if you delete the line no test will fail, if you delete the file no test will probably fail 🤷 

there is no side-effect of importing `/@embroider/virtual/compat-modules` any more so just adding a script will do nothing to any app ever

Reply:
should we delete the whole dummy folder?

I can't leave @embroider/core/entrypoint because it doesn't exist anymore